site/content/docs/cicd/settings.md

```diff
@@ -0,0 +1,35 @@
+# Deployment Settings File
+
+You can select deployment options interactively or specify them in a [JSON configuration file](docs/features/config-file.md). You can also keep the default values that the tool selects for you.
+
+This section defines the JSON definitions and syntax that you construct and use a deployment settings file. Here is the [JSON file definition](https://github.com/aws/aws-dotnet-deploy/tree/main/src/AWS.Deploy.Recipes/RecipeDefinitions).
+
+The Deployment Settings file is a JSON file that contains all of the settings that the deployment tool uses to drive the experience. The deployment tool uses this file to determine which deployment services to use, and how to configure them. --apply option on deploy command allows you to specify a deployment settings file.
```

Comment:
Minor, but something like "**The** `--apply` option..." sounds nicer

Reply:
fixed